man/tpm2_selftest.1.md

```diff
@@ -0,0 +1,48 @@
+% tpm2_selftest(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2019
+
+# NAME
+
+**tpm2_selftest**(1) - Run TPM's self-test internal routines
+
+# SYNOPSIS
+
+**tpm2_selftest** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_selftest**(1) Cause the TPM to execute self-test of its capabilities.
+
+Self-test can be executed in two modes :
+
+* Simple test - TPM will test functions that require testing
+* Full test - TPM will test all functions regardless of what has already been tested
+
+# OPTIONS
+
+* **-F**, **--fulltest** : Run self-test in full mode
```

Comment:
`-f` or `-t` I would take. We want to leave capital arguments for parent object arguments in commands as we work towards 4.0 release. There are still a lot of places we need to fix those, but I don't want to add anymore.

Reply:
Ok on option naming policy ! :+1:

I'll change it to '-f'